packages/components/src/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,122 @@
+import { useContext, useRef } from 'react';
+import type { DialogProps } from 'react-aria-components';
+import { Dialog, OverlayTriggerStateContext } from 'react-aria-components';
+import type { AriaLandmarkRole } from '@react-aria/landmark';
+import { useLandmark } from '@react-aria/landmark';
+import { cn, useClassNames } from '@marigold/system';
+import { DrawerContext } from './Context';
+import { DrawerActions } from './DialogActions';
+import { DrawerContent } from './DrawerContent';
+import { DrawerModal } from './DrawerModal';
+import { DrawerTitle } from './DrawerTitle';
+import { DrawerTrigger } from './DrawerTrigger';
+
+// CLose Button
+// ---------------
+interface CloseButtonProps {
+  className?: string;
+}
+
+const CloseButton = ({ className }: CloseButtonProps) => {
```

Comment:
why not reuse the `CloseButton` from `Dialog`?

Reply:
Because it is the same. Ideally one day we just have one button for both.

Reply:
when it is the same than you can reuse it?